### PR TITLE
Enhanced the profile page

### PR DESCRIPTION
**Pull request by arungoel123456**
Fixes #0000 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] PR is descriptively titled 📑
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by arungoel123456** [comment]
mentors please check

**Comment by gauravano** [comment]
Hi @arungoel123456, please add the issue number in the main PR comment as `Fixes #[issue-number]` to let us know the issue you are solving and also this way issue will be closed automatically once PR gets merged. Thanks!

**Comment by gauravano** [comment]
Also, please remove chnages in gemfile, gemfile.lock, and yarn.lock from PR. And, as this seems to be design related issue so please add screenshot of updated view. Thanks!

**Comment by arungoel123456** [comment]
@gauravano I will do it today .

**Comment by gauravano** [comment]
Great!

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.4.4'
+ruby '2.3.3'
```
Comment (namangupta01) [inline Gemfile]: Change this back to 2.4.4

**Comment by arungoel123456** [comment]
I will do it today

**Comment by SidharthBansal** [comment]
Please rebase this PR as there are conflicts.

**Comment by gauravano** [comment]
Hi @arungoel123456, as there's no update on this PR, so closing this PR for now. Feel free to reopen it and complete it if you're still interested. Thanks!

**Pull request closed without merging**